src/services/user.js

```diff
@@ -0,0 +1,4 @@
+export const getFullName = user =>
+  [user.properties.first_name, user.properties.last_name].join(' ');
```

Comment:
```suggestion
  [user.properties.first_name, user.properties.last_name].join(' ').trim();
```
To avoid a space if one or other prop isn't filled

Reply:
Fixed